### PR TITLE
update: outdated integration docs, simplify l1 and l2 costs explanation

### PR DESCRIPTION
**Pull request by platocrat**

### CODE REVIEW

File: src/docs/developers/integration.md [hunk 1]
```diff
@@ -54,18 +54,19 @@ The first part of getting started with Optimistic Ethereum is to get your contra
...
 
-If you want to try out an example contract instead of deploying your own, you can follow our [tutorial](https://github.com/ethereum-optimism/optimism-tutorial/blob/main/README.md) (which should be pretty seamless).
+If you want to try out an example contract instead of deploying your own, you can follow our [tutorial](https://github.com/ethereum-optimism/optimism-tutorial) (which should be pretty seamless).
 If you're using another testing suite like [truffle](https://www.trufflesuite.com/), that tutorial won't apply. But these `npm` packages have got you covered:
 
 - [`@eth-optimism/solc`](https://www.npmjs.com/package/@eth-optimism/solc): exports the Optimistic Ethereum compiler for `solidity@0.5/0.6/0.7`
-- [`@eth-optimism/plugins`](https://www.npmjs.com/package/@eth-optimism/plugins): exports "OVM-ified" `waffle.mockProvider` and `ganache` packages which will work with contracts output by the compiler.
+- [`@eth-optimism/hardhat-ovm`](https://www.npmjs.com/package/@eth-optimism/hardhat-ovm): exports "OVM-ified" `artifacts-ovm` folder of contract artifacts which will work with contracts output by the compiler.
```
Comment (snario) [inline src/docs/developers/integration.md:67]: We may say "OVM-ified" colloquially internally but or a newcomer reading these docs, it is not clear what this implies.

File: src/docs/developers/integration.md [hunk 2]
```diff
@@ -54,18 +54,19 @@ The first part of getting started with Optimistic Ethereum is to get your contra
...
-2. Deploy and test your contracts onto the local L2 node using `optimism-integration`.
+2. Deploy and test your contracts onto the local L2 node using our [`optimism`](https://github.com/ethereum-optimism/optimism#integration-tests) monorepo.
 
-If you're using [hardhat](https://hardhat.org), our preferred development environment, this can all be done with the use of some handy plugins we've built.
+If you're using [hardhat](https://hardhat.org), our preferred development environment, this can easily be done with the use of a handy plugin we've built.
 For the rest of this documentation, we'll expect you to have your Solidity contracts ready to go.
 Please note that Vyper support is currently not enabled but is planned for a future release.
 
-If you want to try out an example contract instead of deploying your own, you can follow our [tutorial](https://github.com/ethereum-optimism/optimism-tutorial/blob/main/README.md) (which should be pretty seamless).
+If you want to try out an example contract instead of deploying your own, you can follow our [tutorial](https://github.com/ethereum-optimism/optimism-tutorial) (which should be pretty seamless).
```
Comment (snario) [inline src/docs/developers/integration.md:63]: Unnecessary parentheses.

File: src/docs/developers/integration.md [hunk 3]
```diff
@@ -54,18 +54,19 @@ The first part of getting started with Optimistic Ethereum is to get your contra
 This process involves two primary sub-steps:
 
 1. Compile your contracts with the OVM Solidity compiler.
-2. Deploy and test your contracts onto the local L2 node using `optimism-integration`.
+2. Deploy and test your contracts onto the local L2 node using our [`optimism`](https://github.com/ethereum-optimism/optimism#integration-tests) monorepo.
 
-If you're using [hardhat](https://hardhat.org), our preferred development environment, this can all be done with the use of some handy plugins we've built.
+If you're using [hardhat](https://hardhat.org), our preferred development environment, this can easily be done with the use of a handy plugin we've built.
```
Comment (snario) [inline src/docs/developers/integration.md:59]: Might be unnecessary to say "easily". Makes it frustrating if it doesn't work :p

**Review by snario — APPROVED** [review]
Good additions. We ought to improve our documentation to recommend using a Docker image for local tests, not to clone the repository.

**Comment by smartcontracts** [comment]
> We ought to improve our documentation to recommend using a Docker image for local tests, not to clone the repository.

Agree, just spoke with @gakonst about this and we're going to have published images soon that'll remove the need to clone the repo. But for now this is the only option, might as well document it.

**Review by smartcontracts — APPROVED** [review]

**Pull request merged**